### PR TITLE
[fix](inverted index) warm up inverted index

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/cloud/cloud_warm_up_manager.cpp [hunk 1]
```diff
@@ -114,6 +114,40 @@ void CloudWarmUpManager::handle_jobs() {
...
+                                        [wait](Status st) {
+                                            if (!st) {
+                                                LOG_WARNING("Warm up error ").error(st);
+                                            }
+                                            wait->signal();
+                                        },
+                        };
+                        _engine.file_cache_block_downloader().submit_download_task(std::move(meta));
+                    };
+                    auto idx_version = rs->tablet_schema()->get_inverted_index_storage_format();
```
Comment (csun5285) [inline be/src/cloud/cloud_warm_up_manager.cpp:138]: if (rs->tablet_schema()->has_inverted_index())

File: be/src/cloud/cloud_warm_up_manager.cpp [hunk 2]
```diff
@@ -114,6 +114,40 @@ void CloudWarmUpManager::handle_jobs() {
                                                 wait->signal();
                                             },
                             });
+
+                    // idx
+                    auto download_idx_file = [&](const io::Path& idx_path) {
```
Comment (csun5285) [inline be/src/cloud/cloud_warm_up_manager.cpp:119]: The `CloudTablet::add_rowsets` also requires this.

**Review by csun5285 — CHANGES_REQUESTED** [review]

File: be/src/io/cache/block_file_cache_downloader.h [hunk 3]
```diff
@@ -76,7 +76,7 @@ class FileCacheBlockDownloader {
 
 private:
     void download_file_cache_block(const DownloadTask::FileCacheBlockMetaVec&);
-    void download_segment_file(const DownloadFileMeta&);
+    void download_file(const DownloadFileMeta&);
```
Comment (gavinchou) [inline be/src/io/cache/block_file_cache_downloader.h:79]: why rename?

**Comment by zzzxl1993** [comment]
run buildall

**Review by csun5285 — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**